exercises/nodejs/src/game.js

```diff
@@ -19,124 +22,114 @@ var Game = function () {
   }
 
   var currentCategory = function () {
-    if (places[currentPlayer] == 0) { return 'Pop' }
-    if (places[currentPlayer] == 4) { return 'Pop' }
-    if (places[currentPlayer] == 8) { return 'Pop' }
-    if (places[currentPlayer] == 1) { return 'Science' }
-    if (places[currentPlayer] == 5) { return 'Science' }
-    if (places[currentPlayer] == 9) { return 'Science' }
-    if (places[currentPlayer] == 2) { return 'Sports' }
-    if (places[currentPlayer] == 6) { return 'Sports' }
-    if (places[currentPlayer] == 10) { return 'Sports' }
-    return 'Rock'
+    switch(places[currentPlayer]){
+      case 0: case 4: case 8: return 'Pop'
+      case 1: case 5: case 9: return 'Science'
+      case 2: case 6: case 10: return 'Sports'
+      default: return 'Rock';
```

Comment:
```suggestion
if (places[currentPlayer] % 4 === 0) { return 'Pop' }
    if (places[currentPlayer] % 4 === 1) { return 'Science' }
    if (places[currentPlayer] % 4 === 2) { return 'Sports' }
    if (places[currentPlayer] % 4 === 3) { return 'Rock' }
```

Reply:
I like this suggestion, thank you.

I'll resolve this in my local ide!

Reply:
I think there’s an interesting trade-off between the reduction of measured complexity and the fact that it’s now a little harder to understand how the board is laid out (due to the use of modulo). I’m not a huge fan of switches but I actually think that the switch made it clearer to think about the board. You might be interested to hear that there is a way to still very clearly represent the way the board is laid out, and have an even lower complexity score (of 1). Bit of a challenge 😃

Reply:
I've since refactored this section of code, by defining a `board` variable:

`const board = ['Pop', 'Science', 'Sports', 'Rock','Pop', 'Science', 'Sports', 'Rock','Pop', 'Science', 'Sports', 'Rock']`

And the `currentCategory` looks like this:

`var currentCategory = function() {
      return board[players[currentPlayer].place]
}`

Was this the one-line solution that you had in mind?